src/service/metrics/otlp_http.rs

```diff
@@ -816,17 +806,13 @@ fn process_hist_data_point(
     }
     process_exemplars(rec, data_point);
 
-    // add count record
-    let mut count_rec = rec.clone();
-    count_rec[VALUE_LABEL] = json::get_float_value(data_point.get("count").unwrap()).into();
-    count_rec[NAME_LABEL] = format!("{}_count", count_rec[NAME_LABEL].as_str().unwrap()).into();
-    bucket_recs.push(count_rec);
-
-    // add sum record
-    let mut sum_rec = rec.clone();
-    sum_rec[VALUE_LABEL] = json::get_float_value(data_point.get("sum").unwrap()).into();
-    sum_rec[NAME_LABEL] = format!("{}_sum", sum_rec[NAME_LABEL].as_str().unwrap()).into();
-    bucket_recs.push(sum_rec);
+    // add 4 types of record
+    for stream in ["count", "sum", "min", "max"] {
+        let mut new_rec = rec.clone();
+        new_rec[VALUE_LABEL] = json::get_float_value(data_point.get(stream).unwrap()).into();
```

Comment:
we can assume the `min` and `max` always exist, if not the `unwrap` will panic, we should check it.

Reply:
Updated: To keep it consistent with gPRC case, added  "null"  when non-existing. e.g.
```
{"__name__": String("test_count"),  "_timestamp": Number(0), "value": Null}
```